obspy/io/seiscomp/sc3ml.py

```diff
@@ -383,6 +383,9 @@ def _read_channel(inventory_root, cha_element, _ns):
                                 unit=True)
     depth = _read_floattype(cha_element, _ns("depth"), Distance,
                             unit=True)
+    # Set depth to 0 if it is missing (#1816)
+    if depth is None:
+        depth = 0
```

Comment:
Same goes for longitude/latitude/elevation, they're all not mandatory in sc3ml according to the schema..

Also, please show a warning message, e.g.
```python
    if depth is None:
        msg = "Channel is missing depth information, using '0.0'"
        warnings.warn(msg)
        depth = 0
```